packages/kit/src/vite/dev/index.js

```diff
@@ -250,9 +290,12 @@ export async function dev(vite, vite_config, svelte_config, illegal_imports) {
 					);
 				}
 
-				const runtime_base = process.env.BUNDLED
-					? `/${posixify(path.relative(cwd, `${svelte_config.kit.outDir}/runtime`))}`
-					: `/@fs${runtime}`;
+				// For some reason using runtime_prefix here is buggy, since Vite will later load the modules
+				// again with a slightly different url (with the drive letter) on windows
+				const runtime_base = `/@fs${
+					// Windows/Linux separation - Windows starts with a drive letter, we need a / in front there
+					runtime_directory.startsWith('/') ? '' : '/'
+				}${runtime_directory}`;
```

Comment:
This was needed for some reason - makes me uneasy and I don't know why it's needed (only?) in this place. Do Vite residents @bluwy and @dominikg know more?

Reply:
there is something similar in v-p-s codebase for the virtual css modules.. i don't remember what exactly in vite made in neccessary. https://github.com/sveltejs/vite-plugin-svelte/blob/ed2a22d0e411b71d39311a5026b3c72762e90e4b/packages/vite-plugin-svelte/src/utils/id.ts#L65

Reply:
i'm gonna go with 'if it works it works' 😆